libs/ledger-live-common/src/families/stacks/bridge/utils/misc.ts

```diff
@@ -206,6 +263,7 @@ export const getAccountShape: GetAccountShape = async info => {
     balance,
     spendableBalance,
     operations: flatMap(rawTxs, mapTxToOps(accountId)),
+    pendingOperations: flatMap(mempoolTxs, mapPendingTxToOps(accountId, address)),
```

Comment:
we shouldn't add pendingOperations here.
This pendingOperations is used for new broadcasted transaction from our UI. (not even necessarily in mempool, but we want to display them to users in our Ledger live)
what we should do is to merge  ```flatMap(rawTxs, mapTxToOps(accountId))``` and ```flatMap(mempoolTxs, mapPendingTxToOps(accountId, address))```
in the above "operations" field. for confirmed transaction, we set "blockHeight" and "blockHash" field correctly and for pending transaction in mempool, we set "blockHeight" and "blockHash" field as null. So that we can see these pending transactions in our Ledger live
So that